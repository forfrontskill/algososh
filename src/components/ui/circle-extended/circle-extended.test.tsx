import CircleExtended from "./circle-extended";
import renderer from 'react-test-renderer';

describe('Test CircleExtended component', () => {

    it('CircleExtended rendering', () => {
        const tree = renderer
            .create(<CircleExtended main={{index:0}} />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});