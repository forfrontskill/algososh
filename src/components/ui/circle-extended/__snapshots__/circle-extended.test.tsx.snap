// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test CircleExtended component CircleExtended rendering 1`] = `
Array [
  <div
    className="SmallCircleContainer"
  />,
  <div
    className="CircleExtended"
  >
    <div
      className="CircleWirhArrow"
    >
      <div
        className="content MainCircle"
        data-testid="circle"
      >
        <div
          className="text text_type_input text_color_input mb-4 absolute head element"
          data-testid="circle-head"
        />
        <div
          className="circle   default"
          data-testid="circle-main"
        >
          <p
            className="text text_type_circle text_color_input letter"
            data-testid="circle-text"
          />
        </div>
        <p
          className="text text_type_input text_color_input mt-4 absolute index"
          data-testid="circle-index"
        >
          0
        </p>
        <div
          className="text text_type_input text_color_input mt-4 absolute tail60 element"
          data-testid="circle-tail"
        />
      </div>
      <div
        className="Arrow"
      />
    </div>
    <div
      className="SmallCircleContainer"
    />
  </div>,
]
`;
